docs/contributing/in_progress/index.rst

```diff
@@ -0,0 +1,76 @@
+.. _in-progress:
+  
+********************
+Features In-Progress
+********************
+
+This is a current status summary of ongoing development projects in the TARDIS collaboration. Brief summaries of the projects are listed below. Other projects and their descriptions can be found in the `GitHub Projects <https://github.com/tardis-sn/tardis/projects>`_ or the `TARDIS Enhancement Proposals (TEPS) repository <https://github.com/tardis-sn/tep>`_.
+
+Additionally, documentation for features that are not yet implemented can be found here.
+
+
+Google Summer of Code (GSOC)
+============================
+
+Many new TARDIS developments are made through GSOC students. To learn more about potential GSOC projects, see the `GSOC ideas page <https://tardis-sn.github.io/gsoc/tardis-gsoc-ideas.html>`_.
+
+
+
+Restructure
+===========
+
+Restructuring TARDIS to be more modularized in order to facilitate the addition of new physics modules and improve maintainability. See https://github.com/orgs/tardis-sn/projects/5.
```

Comment:
This URL is to a private project board, so you should remove it

Reply:
Ahh, I didn't realize it was private